src/Symfony/Component/Messenger/Adapter/PhpAmqp/AmqpFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Adapter\PhpAmqp;
```

Comment:
Why did you use this namespace `PhpAmqp` ? 
I guess it's because it's for the  PHP AMQP PECL
I said that because it could be confused with https://github.com/php-amqplib/php-amqplib

I would named it `Amqp`

Reply:
Fair enough, this could be confused, but `Amqp` is as vague. What about `AmqpExt`?